.travis.yml

```diff
@@ -1,31 +1,48 @@
-# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
-
-language: R
+language: r
 dist: bionic
-sudo: false
-cache: packages
-latex: true
-r_github_packages: mlr-org/mlr3book
-pandoc_version: 2.7.3
+cache:
+  - packages
+  - ccache
```

Comment:
Why do we need ccache now? We are hopefully not compiling anything expensive now to build the book?

Reply:
This is just for a fast installation of dependencies for the dep installation process.

We used this since months in many repos and for > 1 year in _mlr_.